tests/mainnet.rs

```diff
@@ -57,43 +51,21 @@ fn mainnet_blocks_from_rpc() {
         common::test_execute_alloy(&rpc_storage, &chain, block.clone(), true);
 
         // Snapshot blocks (for benchmark)
-        // TODO: Port to a dedicated CLI instead?
         // TODO: Binary formats to save disk?
         if std::env::var("SNAPSHOT_BLOCKS") == Ok("1".to_string()) {
-            let dir = format!("data/blocks/{block_number}");
-            fs::create_dir_all(dir.clone()).unwrap();
-            let file_block = File::create(format!("{dir}/block.json")).unwrap();
-            serde_json::to_writer(file_block, &block).unwrap();
-
-            // TODO: Snapshot with consistent ordering for ease of diffing.
-            // Currently [EvmStorage]'s storage ordering isn't consistent.
-            let mut state = BTreeMap::<Address, EvmAccount>::new();
-            let mut bytecodes: BTreeMap<B256, EvmCode> = match File::open("data/bytecodes.bincode")
-            {
-                Ok(file) => bincode::deserialize_from(BufReader::new(file)).unwrap(),
-                Err(_) => BTreeMap::new(),
-            };
-            bytecodes.extend(rpc_storage.get_cache_bytecodes());
-            for (address, mut account) in rpc_storage.get_cache_accounts() {
-                if let Some(code) = account.code.take() {
-                    assert_ne!(account.code_hash.unwrap(), KECCAK_EMPTY);
-                    bytecodes.insert(account.code_hash.unwrap(), code);
-                }
-                state.insert(address, account);
-            }
-
-            let file_state = File::create(format!("{dir}/pre_state.json")).unwrap();
-            serde_json::to_writer(file_state, &state).unwrap();
-            let file_bytecodes = File::create("data/bytecodes.bincode").unwrap();
-            bincode::serialize_into(file_bytecodes, &bytecodes).unwrap();
-
-            // We convert to [BTreeMap] for consistent ordering & diffs between snapshots
-            let block_hashes: BTreeMap<u64, B256> =
-                rpc_storage.get_cache_block_hashes().into_iter().collect();
-            if !block_hashes.is_empty() {
-                let file = File::create(format!("{dir}/block_hashes.json")).unwrap();
-                serde_json::to_writer(file, &block_hashes).unwrap();
-            }
+            process::Command::new("cargo")
```

Comment:
Let's move this to `README.md` under `Development` instead.

Reply:
Sorry, could you elaborate on this part a bit more? Do you want me to add this command in the MD so users know how to run it themselves?

Reply:
Added to the readme.md https://github.com/risechain/pevm/pull/357/commits/fdaa244b3664d0473a811dd8190b7dd5e96693ff.